src/main/resources/swagger.api/inn-reach.yaml

```diff
@@ -5,175 +5,161 @@ info:
 paths:
-  /configurations:
-    get:
-      responses:
-        '200':
-          description: ""
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/moduleConfigurations"
-        '500':
-          description: Internal server error
-          content:
-            text/plain:
-              schema:
-                type: string
-                example: Internal server error
-      description: Get a list of inn reach configurations
-      operationId: getConfigurations
-      parameters:
-        - $ref: "#/components/parameters/trait_pageable_offset"
-        - $ref: "#/components/parameters/trait_pageable_limit"
-        - $ref: "#/components/parameters/trait_queryable_query"
+  /locations:
     post:
       responses:
         '201':
-          description: ""
+          description: Created
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/moduleConfiguration"
+                $ref: "#/components/schemas/innReachLocationDTO"
         '400':
           description: Bad request
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Bad request
-        '422':
-          $ref: "#/components/responses/trait_validate_422"
+                $ref: "#/components/schemas/validationErrorsDTO"
         '500':
           description: Internal server error
           content:
-            text/plain:
+            application/json:
               schema:
-                type: string
-                example: Internal server error
-      description: Add new inn reach configuration
-      operationId: postConfiguration
+                $ref: "#/components/schemas/error"
+      description: Add new InnReach location
+      operationId: postInnReachLocation
       requestBody:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/moduleConfiguration"
+              $ref: "#/components/schemas/innReachLocationDTO"
         required: true
-    put:
+    get:
       responses:
-        '201':
-          description: ""
+        '200':
+          description: "OK"
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/moduleConfiguration"
-        '400':
-          description: Bad request
+                $ref: "#/components/schemas/innReachLocationsDTO"
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+      description: Get a list of InnReach locations
+      operationId: getLocations
+      parameters:
+        - $ref: "#/components/parameters/trait_pageable_offset"
+        - $ref: "#/components/parameters/trait_pageable_limit"
+  /locations/{locationId}:
```

Comment:
please see how to define a parameter as UUID in Quick MARC [records-editor.yaml](https://github.com/folio-org/mod-quick-marc/blob/762d90d12b41a8c6f076513004b4b52aa5ca6194/src/main/resources/swagger.api/records-editor.yaml#L130)